### PR TITLE
feat: cover ext user flows with notifications and actions

### PR DESCRIPTION
**Pull request by inca**
dac94b3839d6b60421e429a6f3de0b54

There are some corner cases (when versions don't meet the requirements) which I implemented but haven't tested just yet. Will do some more testing tomorrow — but should be good to review as it is.

### CODE REVIEW

**Comment by inca** [comment]
Found out that script is not properly initialized when loaded via `loadAutomationJson` (i.e. no extensions checked/installed).

This will need more work, so converting to draft.

**Comment by inca** [comment]
Ok, should be good now.

File: packages/autopilot/src/app/controllers/notifications.ts [hunk 1]
```diff
@@ -30,30 +30,34 @@ export class NotificationsController {
...
+            level,
             style: 'float',
             canClose: true,
             timeout: 0,
             ...spec,
         };
     }
 
+    protected matchById(notification: Notification, id: string) {
+        const pattern = new RegExp('^' + id.replace(/\./g, '\\.').replace(/\*/g, '.*') + '$');
```
Comment (kyungeunni) [inline packages/autopilot/src/app/controllers/notifications.ts:59]: 👍

**Review by kyungeunni — APPROVED** [review]
LGTM ✨

**Pull request merged**